website/vercel.json

```diff
@@ -2,6 +2,11 @@
   "cleanUrls": true,
   "trailingSlash": false,
   "redirects": [
+    {
+      "source": "/docs/cloud/about-cloud/regions-ip-addresses",
+      "destination": "/docs/cloud/about-cloud/accesss-regions-ip-addresses",
+      "permanent": true
```

Comment:
there are two older redirects related to `regions-ip-addresses`:
- https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/vercel.json#L1696
- https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/vercel.json#L1321

i might be misremembering, was there a recent decision on how we should handle older redirects for a page?